features/step_definitions/bootstrap_steps.rb

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2010-2017 SUSE-LINUX
+# Licensed under the terms of the MIT license.
+
+And (/^I create the bootstrap-repo for "([^"]*)" on the "([^"]*)"$/) do |target, server|
+  node = get_target(target)
+  os_version_raw, _code = node.run('grep "VERSION=" /etc/os-release')
+  os_version = os_version_raw.strip.split('=')[1].delete '"'
+  puts 'i should see the os version: ' + os_version
+  command = "mgr-create-bootstrap-repo -c SLE-" + os_version + "-x86_64"
+  puts 'running the command on the server:' + command
+  _out, $fail_code = get_target(server).run(command, false)
```

Comment:
here i would just use: 
```$server.run(command, false)```
since $server is a globall variable that we define in ```support`` dir. Every file that we have there, classes, function, vars, is automatically avaible for the feature and steps.